### PR TITLE
Add ScholarshipPoints

### PR DESCRIPTION
**Pull request by AverageHelper**
This deletion process is a bit of a runaround, but their policies and confirmation emails indicate a reasonable erasure of personal identifying data.

### CODE REVIEW

**Comment by AverageHelper** [comment]
Got another one ready for ya

File: _data/sites.json [hunk 1]
```diff
@@ -8834,6 +8834,17 @@
         ]
     },
 
+    {
+        "name": "ScholarshipPoints",
```
Comment (tupaschoal) [inline _data/sites.json:8838]: Should the name be Revisões/ScholarshipPoints? I don't know the service, so I can't tell the relationship between the two

Comment (AverageHelper) [reply _data/sites.json:8838]: The parent company is Edvisors. I can add them to the name, but I haven't heard of Revisões.

Comment (tupaschoal) [reply _data/sites.json]: It was my silly autocorrect, the intention was to write Edvisors :P

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**